openedx/core/djangoapps/user_authn/waffle.py

```diff
@@ -0,0 +1,13 @@
+"""
+Feature toggles for user_authn.
+"""
+from openedx.core.djangoapps.waffle_utils import WaffleFlagNamespace, WaffleFlag
+
+# Namespace
+_WAFFLE_NAMESPACE = u'user_authn'
+_WAFFLE_FLAG_NAMESPACE = WaffleFlagNamespace(_WAFFLE_NAMESPACE)
+
+# Flags
+
+# Intended as a temporary toggle for roll-out of jwt sessions cookies feature.
+JWT_SESSION_COOKIES_FLAG = WaffleFlag(_WAFFLE_FLAG_NAMESPACE, u'jwt_session_cookies')
```

Comment:
Although this is only a temporary switch, can we remove "session" from the name and value?  By proper definition, "session cookies" are always deleted when the browser's session is closed.  But for us, we may have expiration times that surpass the browser session - making them "permanent cookies" instead.

Reply:
Sounds good.  I took this from "Authn: JWT Session Cookies" which was the name of our Story.  I updated the story to be called "Authn: JWT Cookies" as well.